pilot/pkg/networking/plugin/mixer/mixer.go

```diff
@@ -49,13 +49,62 @@ const (
 
 	// defaultConfig is the default service config (that does not correspond to an actual service)
 	defaultConfig = "default"
+
+	// disablePolicy check
```

Comment:
cc @kyessenov / @douglas-reid / @mandarjog .. please check the logic here and ack if okay.